api/v1alpha3/virtualmachine_types.go

```diff
@@ -449,6 +449,12 @@ type VirtualMachineSpec struct {
 	// +optional
 	// +kubebuilder:validation:Format:=uuid
 
+	// InstanceUUID describes the desired Instance UUID for a VM.
+	// If omitted, this field defaults to a random UUID.
+	InstanceUUID string `json:"instanceUUID,omitempty"`
+	// +optional
```

Comment:
Please leave an empty line between the field and the annotations for the next field.